xml/admin_administration.xml

```diff
@@ -205,40 +205,45 @@
   <sect2 xml:id="commands.node.managment.issue-generator">
    <title>The <command>issue-generator</command> Command</title>
    <para>
-    The <command>issue-generator</command> creates a volatile temporary
-    <filename>/run/issue</filename> file. The file
-    <filename>/etc/issue</filename> should be a symbolic link to the temporary
-    <filename>/run/issue</filename>.
+    On &suse; &mos;, <filename>/etc/issue</filename> is generated by <command>issue-generator</command>.
+   </para>
+   <para>The <command>issue-generator</command> program allows &mos; to display
+    a dynamic message above the login prompt, which is generated from the
+    contents of several different files.
    </para>
    <para>
-    You can use the command to prefix all directories and files with a
-    specified prefix (PATH in this example):
+    Files in <filename>/etc/issue.d</filename> override files with the same name in
```

Comment:
I think you should start with mentioning that /etc/issue is only a symbolic link to /run/issue, which is generated dynamically by issue-generator. Thus, users must not touch these two files.

Next you would mention that they should make changes or additions in /etc/issue.d, after that you would continue with /usr/lib/issue.d and eventually mention /run/issue.d.